doonut-app-external-api/src/main/kotlin/com/doonutmate/calendar/controller/CalendarController.kt

```diff
@@ -55,4 +58,32 @@ class CalendarController(
         calendarAppService.report(CalendarReportReason.of(memberId, req.calendarId, req.reason))
         return ResponseEntity.ok().build()
     }
+
+    @Operation(summary = "캘린더 조회", description = "멤버가 자신의 캘린더 제목을 조회한다.(토큰사용)")
```

Comment:
summary하고 description 정보가 같다면 summary만 사용해도 될 것 같아요. 
```suggestion
    @Operation(summary = "캘린더 제목 조회")
```